CHANGELOG.md

```diff
@@ -8,37 +8,92 @@ Changelog
   * acl (@saqibraza)
   * ace (@yjyongz)
   * remark ace (@bansalpradeep)
+  * **Platforms:** Nexus (3|9)k
```

Comment:
I'd prefer `Nexus 3k/9k` or even `N3k/N9k` - if I want to grep through this file to find which properties are supported on my platform, I can't easily do so if sometimes it's `3k` and other times it's `(3|`. Better to have a consistent string used throughout.